orchagent/montxorch.cpp

```diff
@@ -0,0 +1,251 @@
+#include <linux/if_ether.h>
+
+#include <unordered_map>
+#include <utility>
+#include <exception>
+#include <vector>
+
+#include "montxorch.h"
+#include "orch.h"
+#include "port.h"
+#include "logger.h"
+#include "sai_serialize.h"
+#include "converter.h"
+#include "portsorch.h"
+
+extern sai_port_api_t *sai_port_api;
+extern PortsOrch*       gPortsOrch;
+using namespace std::rel_ops;
+
+const std::string currentDateTime() 
+{
+    time_t     now = time(0);
+    struct tm  tstruct;
+    char       buf[80];
+    tstruct = *localtime(&now);
+    strftime(buf, sizeof(buf), "%Y-%m-%d.%X", &tstruct);
+    return buf;
+}
+
+MonTxOrch::MonTxOrch(TableConnector appDb, TableConnector confDb, TableConnector stateDb):
+    Orch(confDb.first,confDb.second),
+    m_TxErrorTable(appDb.first,appDb.second),
+    m_stateTxErrorTable(stateDb.first,stateDb.second),
+    m_pollPeriod(0)
+{
+    SWSS_LOG_ENTER();    
+    m_pollTimer = new SelectableTimer(timespec { .tv_sec = 0, .tv_nsec = 0 });
+    Orch::addExecutor( new ExecutableTimer(m_pollTimer, this, TXPORTMONORCH_SEL_TIMER));
+    SWSS_LOG_NOTICE("MonTxOrch initialized with the following tables %s %s \n",stateDb.second.c_str(),appDb.second.c_str());
+}
+
+void MonTxOrch::startTimer(uint32_t interval)
+{
+    SWSS_LOG_ENTER();
+    try
+    {
+        auto timespecInterval = timespec { .tv_sec = interval , .tv_nsec = 0 };
+        SWSS_LOG_INFO("MonTxOrch::startTimer with executor: %p\n", m_pollTimer);
```

Comment:
You don't need to specify the function name in each log message. It is the responsibility of the log macro. Please remove it from all the messages.